includes/REST_API/Stories_Media_Controller.php

```diff
@@ -300,7 +304,7 @@ protected function prepare_items_query( $prepared_args = [], $request = null ) {
 	 *
 	 * @param WP_Post         $post    Attachment object.
 	 * @param WP_REST_Request $request Request object.
-	 * @return WP_REST_Response Response object.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
```

Comment:
It was always possible this was `WP_Error`, because of the `web_stories_rest_prepare_attachment` filter. 

I am just making clear, it could either be a `WP_REST_Response` or `WP_Error`. If anything that is returned from that filter, that it will just break everything.

Reply:
Since a filter could return literally anything, wouldn't it make more sense for this to be `mixed`?

But I am actually -1 on doing this change here.

`prepare_item_for_response` and `web_stories_rest_prepare_attachment` are not supposed to return errors. They expect proper response objects.

Let's keep the PHPDoc at `@return WP_REST_Response` and ignore any PHPCS warnings about missing return type hints.

Reply:
Okay, I have reverted the changes and added back the type hinting. There is type hint makes sense here. If you return anything other than a `WP_REST_Response`, it will fail anyway. You agree?